src/sentry/static/sentry/app/views/integrationInstallation.jsx

```diff
@@ -1,112 +1,169 @@
 import React from 'react';
+import styled from 'react-emotion';
 
+import {singleLineRenderer} from 'app/utils/marked';
+import {t, tct} from 'app/locale';
+import AddIntegration from 'app/views/organizationIntegrations/addIntegration';
+import Alert from 'app/components/alert';
 import AsyncView from 'app/views/asyncView';
-import Form from 'app/views/settings/components/forms/form';
+import Button from 'app/components/button';
+import Field from 'app/views/settings/components/forms/field';
+import HookStore from 'app/stores/hookStore';
 import IndicatorStore from 'app/stores/indicatorStore';
 import NarrowLayout from 'app/components/narrowLayout';
-import SelectField from 'app/views/settings/components/forms/selectField';
-import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
-import {t, tct} from 'app/locale';
-
-class IntegrationInstallation extends AsyncView {
-  componentDidMount() {
-    this.dialog = null;
-    window.addEventListener('message', this.receiveMessage, false);
-  }
-
-  componentWillUnmount() {
-    window.removeEventListener('message', this.receiveMessage);
+import SelectControl from 'app/components/forms/selectControl';
 
-    if (this.dialog !== null) {
-      this.dialog.close();
-    }
-  }
+export default class IntegrationInstallation extends AsyncView {
+  state = {
+    selectedOrg: null,
+    organization: null,
+    providers: [],
+    reloading: false,
+  };
 
   getEndpoints() {
-    return [['organizations', '/organizations/?owner=1']];
+    return [['organizations', '/organizations/']];
```

Comment:
Won't this receive all Orgs, regardless of ownership/membership? ie. Orgs the User has no access to.

Reply:
What do you mean all orgs? Like as in _all_ orgs entirely? 

https://github.com/getsentry/sentry/blob/230f91904e4b596f652e62333a0b013866773bf2/src/sentry/api/endpoints/organization_index.py#L61-L64

Reply:
Where does it actually apply that logic, though? I'm probably missing something, but all I see it doing in this case is `Organization.objects.distinct()` and then serializing that.